setup.py

```diff
@@ -43,40 +43,27 @@
         "see**\n`here <https://pypi.org/project/cirq>`__.\n\n" + long_description
     )
 
-# Read in requirements
-requirements = open('requirements.txt').readlines()
-requirements = [r.strip() for r in requirements]
-contrib_requirements = open('cirq/contrib/contrib-requirements.txt').readlines()
-contrib_requirements = [r.strip() for r in contrib_requirements]
-dev_requirements = open('dev_tools/conf/pip-list-dev-tools.txt').readlines()
-dev_requirements = [r.strip() for r in dev_requirements]
-
-cirq_packages = ['cirq'] + ['cirq.' + package for package in find_packages(where='cirq')]
-
 # Sanity check
 assert __version__, 'Version string cannot be empty'
 
+# This is a pure metapackage that installs all our packages
+requirements = [f'{p}=={__version__}' for p in ['cirq-core', 'cirq-google']]
```

Comment:
This assumes that `cirq-core` and `cirq-google` will always have the same version. Is this a valid assumption to make?

Reply:
This is stronger than an assumption - it's by design. This is how we "simulate" the monorepo structure. Unless we have good reasons to do otherwise, all modules are going to be released simultaneously and cirq vx.y.z will always pull in cirq-* x.y.z. 

This expectation is captured here and if the design changes for any of our modules, we can change the test as well.